### DISCUSSION TITLE
Is there a way to disable verification emails for login?

### DISCUSSION THREAD

**Discussion by StudioNirin** [lukevella/rallly#850] [General]
Having to get an email code every time you login is pretty... unnecessary. For the first sign up sure,makes sure it's not spam, but after that a saved password is more than sufficient.

Is it not possible to set up this way? May disable the requirement for logins otherwise as it'll make users lives easier probably to just do everything as guest. Seems a waste though.

**Comment by lukevella** [comment]
Sessions are set up to last 30 days so you should only really have to log in once a month. Is that not the case for you?

If you're self-hosting you probably don't want anyone to be able to just use your instance so requiring authentication seems sensible to me.
